### PR TITLE
tmux_cheat_sheet: Refactor json data

### PR DESCRIPTION
**Pull request by NaveenKarippai**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [ ] New Instant Answer
    - [ ] Cheat Sheets: **`New {Cheat Sheet Name} Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [x] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [x] Enhancement: **`tmux_cheat_sheet: Refactor json data`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

The cheatsheet should be refactored to follow Docs cheatsheet
guidelines with some minor modifications on commands included.

Provide an overview of the changes this pull request introduces.

* refactored tmux cheatsheet json data

###### Which issues (if any) does this fix?

Fixes #NNNN - how specifically does it fix the issue?

###### People to notify (@GuiltyDolphin, @moollaza , @zekiel )

---

Instant Answer Page: https://duck.co/ia/view/tmux_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @charles-l

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/tmux.json [hunk 1]
```diff
@@ -11,112 +11,112 @@
...
-            "key": "[Ctrl-B] [Enter]"
+            "key": "[Ctrl] [b], [Enter]"
         }, {
             "val": "Paste selection",
-            "key": "[Ctrl-B] [p]"
+            "key": "[Ctrl] [b], [p]"
         }, {
             "val": "Cursor Up",
-            "key": "[Ctrl-B] [k]"
+            "key": "[Ctrl] [b], [Ctrl] [k]"
```
Comment (charles-l) [inline share/goodie/cheat_sheets/json/tmux.json]: For this one (and the following ones cursor move commands), the key binding is actually [Ctrl] [b], [k]. [Ctrl][b], [ctrl][k] is used for resizing the window.

**Comment by charles-l** [comment]
@NaveenKarippai nice! I made a comment on the diff about some of the keybindings being slightly off, but apart from that it looks good to me. :+1:

File: share/goodie/cheat_sheets/json/tmux.json (hunk 1 above)
Comment (NaveenKarippai) [inline share/goodie/cheat_sheets/json/tmux.json]: @charles-l sorry for that. I have "tmux" configured on my local machine and confused a bit with defaults. It should be good with new changeset.

**Comment by NaveenKarippai** [comment]
@charles-l How about using `[bind-key]` instead of `[Ctrl] [b]` as these are defaults and could vary on  a custom config. We could indicate that `[Ctrl] [b]` is the default `[bind-key]` value on cheatsheet.

**Comment by charles-l** [comment]
@NaveenKarippai hmm... I'm not sure about that. I think the cheatsheet should match the default settings (i.e. the vim cheatsheet is default vim bindings), since someone who has their own customized config likely has a completely different setup anyway (my tmux config is nothing like the default).

**Comment by NaveenKarippai** [comment]
agreed :+1:

**Comment by zekiel** [comment]
thanks for adding some additional insight to this @charles-l

**Pull request merged**